tests/acceptance/features/lib/UsersPage.php

```diff
@@ -184,6 +184,48 @@ public function getEmailOfUser($username) {
 		return $this->getTrimmedText($userEmail);
 	}
 
+	/**
+	 * @return bool
+	 * @throws \Exception
+	 */
+	public function isPasswordFieldOfNewUserVisible() {
+		$newUserPasswordField = $this->findById($this->newUserPasswordFieldId);
+		if ($newUserPasswordField === null) {
+			throw new ElementNotFoundException(
+				__METHOD__ .
+				" $this->newUserPasswordFieldId " .
+				"password field of new user not found"
+			);
+		}
+
+		if (!$newUserPasswordField->isVisible()) {
+			return false;
+		}
+
+		return true;
```

Comment:
```suggestion
		return $newPasswordField->isVisible()
```
isVisible() returns boolean anyway..